scripts/zones/Upper_Jeuno/npcs/_6s2.lua

```diff
@@ -4,11 +4,10 @@
 -- Starts and Finishes Quest: A Clock Most Delicate, Save the Clock Tower, The Clockmaster
 -- !pos -80 0 104 244
 -----------------------------------
-local ID = require("scripts/zones/Upper_Jeuno/IDs")
-require("scripts/settings/main")
-require("scripts/globals/titles")
-require("scripts/globals/shop")
+require("scripts/globals/keyitems")
 require("scripts/globals/quests")
+require("scripts/globals/titles")
+local ID = require("scripts/zones/Upper_Jeuno/IDs")
```

Comment:
Going to argue this pattern, just because how we've started laying out a lot of the Interaction-related items.  I like breaking out locals to their own sub-heading

Reply:
argue all ya want I just want the patter to be constant among our files and not randomly different. mass feex preez.

Reply:
Well, for now, until the style guide is updated, changed it back to local first

Reply:
Just know that anything in `quests/` or `missions/` will be judged differently; outside of that, all good!